.circleci/config.yml

```diff
@@ -1,58 +1,39 @@
-# Python CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-python/ for more details
-#
```

Comment:
Other than neatening, was there any other reason to remove this header? I think it's good to have a human readable section explaining this otherwise cryptic file.